### ISSUE TITLE
Request: JackAss in

### ISSUE THREAD

**Issue by PrinzNika** [falkTX/JackAss#5]
Hi, thanks for the tool, it is really usefull.
I am wondering if it would be possible to also get a midi input version of this? 

I want to use renoise to trigger notes in bitwig and it seems kinda hacky to me, manually opening up alsa midi ports to connect bitwig to, as it only supports that..

cheers,
Nika

**Comment by dromer** [comment]
Yes! Definitely also needs an input version. Not sure how else I could get anything into Bitwig myself either.

**Comment by PowerUser64** [comment]
A way to do something similar to this would be to send JackAss to _MIDI Through 0_ or some other MIDI loopback device, with instruments on different channels.

**Comment by PrinzNika** [comment]
I didn't really understand loopback devices yet. can you elaborate on this?

**Comment by PowerUser64** [comment]
By sending JackAss into a loopback device, your DAW might be able to help you route it places because it should see it as a standard MIDI device.
